2011-04-26  Manjunath Matti  <manjunath81@gmail.com>

	* ports/sysdeps/unix/sysv/linux/arm/eabi/libc-do-syscall.S
	(CALL_MCOUNT): Redefine to empty.

---
 ports/sysdeps/unix/sysv/linux/arm/eabi/libc-do-syscall.S |    3 +++
 1 file changed, 3 insertions(+)

--- a/ports/sysdeps/unix/sysv/linux/arm/eabi/libc-do-syscall.S
+++ b/ports/sysdeps/unix/sysv/linux/arm/eabi/libc-do-syscall.S
@@ -29,6 +29,9 @@
 	.syntax unified
 	.hidden __libc_do_syscall
 
+#undef CALL_MCOUNT
+#define CALL_MCOUNT
+
 ENTRY (__libc_do_syscall)
 	.fnstart
 	push	{r7, lr}
